### PR TITLE
Fix colour scheme and exclude tasks filters for charts

### PR DESCRIPTION
**Pull request by ebadyano**
Fix colour scheme for release-charts and parsing of exclude-tasks to include tags and type filters.

### CODE REVIEW

File: esrally/chart_generator.py [hunk 1]
```diff
@@ -1599,18 +1596,21 @@ def throttled_tasks(self):
...
+                    task_names.append(sub_task.name)
         return task_names
 
 
 def load_race_configs(cfg, chart_type, chart_spec_path=None):
     def add_configs(race_configs_per_lic, flavor_name="oss", lic="oss", track_name=None):
         configs_per_lic = []
         for race_config in race_configs_per_lic:
+            excluded_tasks = None
+            if race_config.get("exclude-tasks") is not None:
```
Comment (danielmitterdorfer) [inline esrally/chart_generator.py:1608]: Let's rewrite this as follows?

```python
if "exclude-tasks" in race_config:
    ...
```

File: esrally/chart_generator.py [hunk 2]
```diff
@@ -1373,12 +1373,13 @@ def load_track(self, cfg, name=None, params=None):
             cfg.add(config.Scope.applicationOverride, "track", "repository.name", self.repository)
             cfg.add(config.Scope.applicationOverride, "track", "track.name", name)
         # another hack to ensure any track-params in the race config are used by Rally's track loader
+        cfg.add(config.Scope.application, "track", "exclude.tasks", excluded_tasks)
```
Comment (danielmitterdorfer) [inline esrally/chart_generator.py:1376]: Can we move this down one line as the comment above does not apply here but rather to the line with track parameters?

**Review by danielmitterdorfer — COMMENTED** [review]
Thanks! Looks mostly fine; I left two suggestions.

**Comment by ebadyano** [comment]
@danielmitterdorfer thank you, I updated the pull request

**Review by danielmitterdorfer — APPROVED** [review]
Thanks! LGTM

**Comment by ebadyano** [comment]
@danielmitterdorfer thank you for the review

**Pull request merged**